tools/licenses/analysis_options.yaml

```diff
@@ -19,7 +19,7 @@ linter:
     - control_flow_in_finally
     - empty_statements
     - hash_and_equals
-    - invariant_booleans
+    # - invariant_booleans # https://github.com/flutter/flutter/issues/5790
```

Comment:
Can you add a couple of words that indicate why it's commented out so everyone doesn't have to click the link to find out?